resource-managers/cook/src/main/scala/org/apache/spark/scheduler/cluster/cook/CoarseCookSchedulerBackend.scala

```diff
@@ -174,12 +174,13 @@ class CoarseCookSchedulerBackend(
   private[this] val jobClient = new JobClient.Builder()
     .setHost(cookHost)
     .setPort(cookPort)
-    .setEndpoint("rawscheduler")
+    .setEndpoint("jobs")
```

Comment:
For Simfarm DFT and ORD, will this change be ok to apply?

Reply:
Yes, /jobs has been around for about 6 months now.